src/shippingModeSelection/shipping-mode-selection.html

```diff
@@ -46,9 +46,7 @@
         <!-- MONDIAL RELAY CHOICE -->
         <div class="animate-mondial-relay mb-3"
              data-ng-if="$ctrl.selectedMode === 'mondialRelay'">
-            <div data-mondial-relay
-                 data-ng-model="$ctrl.selectedRelay">
-            </div>
+            <ng-ovh-mondial-relay data-ng-model="$ctrl.selectedRelay"></ng-ovh-mondial-relay>
```

Comment:
We should figure out if the component should be named `<mondial-relay>` or `<ovh-mondial-relay>`.

ref: https://github.com/ovh-ux/ng-ovh-mondial-relay/pull/20#discussion_r277536260